src/main/resources/alfresco/heartbeat/heartbeat-context.xml

```diff
@@ -20,6 +20,7 @@
         <property name="hbDataSenderService" ref="hbDataSenderService" />
         <property name="scheduler" ref="schedulerFactory" />
         <property name="jobLockService" ref="jobLockService" />
+        <property name="testMode" value="true" />

```

Comment:
Is this correct?  Should this be in test mode by default?

Reply:
This is just to schedule all collectors at 1min intervals, I will remove it before merge